changelogs/fragments/7452-fix-icinga2_host-requiring-ip-key.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is more a feature than a bugfix, since the module didn't claim that it should be possible to use it without the `ip` option before.
```suggestion
minor_changes:
```